assets/clients/components/EditClient.jsx

```diff
@@ -39,6 +39,13 @@ class EditClient extends React.Component {
                                     value={this.props.client.name}
                                     onChange={this.props.onChange}
                                 />
+                                {this.props.client.secret_key &&
```

Comment:
I would add a message there:
`Make sure to copy your new client secret now. You won’t be able to see it again.`

Reply:
and we need to make the client id visible somewhere too

Reply:
Okay, let me add them.